arch/x86/core/intel64/locore.S

```diff
@@ -513,8 +508,14 @@ irq_dispatch:
 	cli
```

Comment:
I don't know where it is exactly, but I know from a previous patch that somewhere around here you have a call to z_check_stack_sentinel().  Note that this can be removed, z_get_next_switch_handle() does this for you (it's the portable EOI hook).

Reply:
Oh, right, will do.  I also need to move RAX to the volatile register set -- I used to "save" and restore it with __swap because it was the return value.  No longer necessary...